src/theme/weave.js

```diff
@@ -144,7 +144,11 @@ const weave = {
     selected: `0px 0px 2px 2px ${colors.primary.lavender}`,
   },
 
-  borderRadius: '4px',
+  borderRadius: {
+    none: '0',
+    soft: '4px',
+    round: '50%',
```

Comment:
what about naming this `circle` instead? that would not only describe to me clearer what the purpose of this radius is but also clarify what the target element is going to look like.